packages/plugin-expo-app/app.js

```diff
@@ -1,11 +1,9 @@
 const Constants = require('expo-constants').default
 const { AppState } = require('react-native')
 
-const appStart = new Date()
```

Comment:
Similar to previous comment I wonder if it would be better to stick with `const appStart = new Date()` at the module level